src/test/java/org/gridsuite/modification/server/service/ModificationRepositoryTest.java

```diff
@@ -701,21 +705,84 @@ public void testSubstationCreation() {
 
         SQLStatementCountValidator.reset();
         networkModificationRepository.deleteModifications(TEST_GROUP_ID, List.of(createSubstationEntity2.getId(), createSubstationEntity3.getId()));
-        assertRequestsCount(2, 0, 1, 2);
+        assertRequestsCount(4, 0, 1, 2);
 
         SQLStatementCountValidator.reset();
         assertEquals(1, networkModificationRepository.getModifications(TEST_GROUP_ID, false, true).size());
-        assertRequestsCount(2, 0, 0, 0);
+        assertRequestsCount(3, 0, 0, 0);
 
         SQLStatementCountValidator.reset();
         networkModificationRepository.deleteModificationGroup(TEST_GROUP_ID, true);
-        assertRequestsCount(2, 0, 0, 3);
+        assertRequestsCount(3, 0, 0, 3);
 
         assertThrows(new NetworkModificationException(MODIFICATION_GROUP_NOT_FOUND, TEST_GROUP_ID.toString()).getMessage(),
             NetworkModificationException.class, () -> networkModificationRepository.getModifications(TEST_GROUP_ID, false, true)
         );
     }
 
+    @Test
+    public void testSubstationCreationWithProperties() {
```

Comment:
Move these tests into `SubstationCreationTest` and `SubstationModificationTest` classes

Reply:
finally I removed them properties are already properly tested in `SubstationCreationTest` and `SubstationModificationTest`